cypress/integration/deals-side-pagination.js

```diff
@@ -0,0 +1,158 @@
+context('Checking deal-group & deal pagination', () => {
+  describe('Pagination', () => {
+    beforeEach(() => {
+      cy.setCookie('kauth_access', Cypress.env('authToken'));
+    });
+    const rowsSelector = 'data-row';
+    let totalNumberOfEntries = 0;
+
+    it('check Pagination for Deal-group dashboard', () => {
+      cy.visit('');
+      cy.url().should('include', 'deal-dashboard/deal-groups');
+      cy.get('.QA-totalPages').then((totalPages) => {
+        totalNumberOfEntries = totalPages.text().split('entries')[0].split('of')[1].replace(' ', '');
+      });
+    });
+
+    it('Verify < arrow is disabled', () => {
```

Comment:
each test should be hyphen with the entity.

i.e. 'Deal-Groups: Verify "<" arrow is disabled'